### DISCUSSION TITLE
Get AWS Backup Vault Size

### DISCUSSION THREAD

**Discussion by swordfish291** [aws/aws-cli#7431] [Q&A] [answered]
Hello Team,

We are tryting to get the size used by our aws backups. We tried looking at cloudwatch metrices but there are no metrics that provide these details. We have multiple accounts and we are trying to find an automated way to get the size used by backups/vaults so that we can look at the overall usage and costs. Is there a way to achieve this using aws cli?

Thank you
Abdul

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
Hi @swordfish291, thanks for reaching out.  The [documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/backup/index.html) indicates that the `describe-backup-vault` method does not have a size parameter in the response.  However, the `describe-backup-job` method does.  I hope that is sufficient for your use case, but if not, I'd be happy to convert this into a feature request.  Please let me know if you want me to do that, or if you have any other questions.
